docs/advanced/advanced_running_stateful_block.md

```diff
@@ -0,0 +1,147 @@
+      persistentvolumeclaim/demo-pvc-raw-block   Bound    pvc-828ce909-6eb2-11ea-abc8-005056a49b44   1Gi
+        
+      ACCESS MODES   STORAGECLASS       AGE
+      RWO            demo-storageclass  78s
+        
+      $> kubectl describe persistentvolume/pvc-828ce909-6eb2-11ea-abc8-005056a49b44
+      Name:            pvc-828ce909-6eb2-11ea-abc8-005056a49b44
+      Labels:          <none>
+      Annotations:     pv.kubernetes.io/provisioned-by: block.csi.ibm.com
+      Finalizers:      [kubernetes.io/pv-protection]
+      StorageClass:    demo-storageclass
+      Status:          Bound
+      Claim:           default/demo-pvc-raw-block
+      Reclaim Policy:  Delete
+      Access Modes:    RWO
+      VolumeMode:      Block
+      Capacity:        1Gi
+      Node Affinity:   <none>
+      Message:
+      Source:
+          Type:              CSI (a Container Storage Interface (CSI) volume source)
+          Driver:            block.csi.ibm.com
+          FSType:            xfs
+          VolumeHandle:      SVC:26;60050760718186998000000000005E93
+          ReadOnly:          false
+          VolumeAttributes:     array_address=demo-management-address
+                                pool_name=demo-pool
+                                storage.kubernetes.io/csiProvisionerIdentity=1631546133261-8081-block.csi.ibm.com
+                                storage_type=SVC
+                                volume_name=demo-prefix_pvc-828ce909-6eb2-11ea-abc8-005056a49b44
+      Events:                <none>
+      </pre>
+
+5. Create a StatefulSet.
+
+    <pre>
+    $> kubectl create -f demo-statefulset-raw-block.yaml
+    statefulset.apps/demo-statefulset-raw-block created
+
+    $> cat demo-statefulset-raw-block.yaml
+        
+    kind: StatefulSet
+    apiVersion: apps/v1
+    metadata:
+      name: demo-statefulset-raw-block
+    spec:
+      selector:
+        matchLabels:
+          app: demo-statefulset
+      serviceName: demo-statefulset
+      replicas: 1
+      template:
+        metadata:
+          labels:
+            app: demo-statefulset
+        spec:
+          containers:
+          - name: demo-container
+            image: registry.access.redhat.com/ubi8/ubi:latest
+            command: [ "/bin/sh", "-c", "--" ]
+            args: [ "while true; do sleep 30; done;" ]
+            volumeDevices:
+              - name: demo-volume-raw-block
+                devicePath: "/dev/block"            
+          volumes:
+          - name: demo-volume-raw-block
+            persistentVolumeClaim:
+              claimName: demo-pvc-raw-block</pre>      
+
+6. Check the newly created pod.
+
+    Display the newly created pod (make sure the pod status is _Running_).
+
+    <pre>
+    $> kubectl get pod demo-statefulset-raw-block-0
+    NAME                 READY   STATUS    RESTARTS   AGE
+    demo-statefulset-raw-block-0   1/1     Running   0          43s
+
```

Comment:
I think there is an extra empty line here and in the end of the block of several steps afterwards as well

Reply:
there doesn't seem to be anything in the source files other than the regular spaces between numbered steps

Reply:
I removed a bunch of lines in this file. please see if that helped or not. if not there isn't anything else i can do